metrics/opencensus/opencensus.go

```diff
@@ -0,0 +1,154 @@
+package opencensus
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"go.opencensus.io/metric/metricdata"
+
+	"github.com/gochain-io/gochain/v3/log"
+	"github.com/gochain-io/gochain/v3/metrics"
+)
+
+// StartExporter launches a goroutine to periodically publish metrics from r to sd.
+func StartExporter(d time.Duration, r metrics.Registry, sd *stackdriver.Exporter) *time.Ticker {
+	t := time.NewTicker(d)
```

Comment:
You don't need the timer, it does this automatically.

Reply:
If we also use the built in stats/collection/views, yes. But not if we need to construct our own custom `metricdata.Metric` as is done here.